docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -204,9 +204,21 @@ Managed
 ~~~~~~~
 
 You can also run acceptance tests on existing "managed" nodes.
-In this case the configuration file should include:
 
-- **addresses**: The IP addresses of two running nodes.
+This is a quicker way to run the acceptance tests as it avoids the slow process of provisioning new acceptance testing nodes.
+
+The ``managed`` provider:
+
+* re-installs the ``clusterhq-flocker-node`` and ``clusterhq-python-flocker`` packages,
+* creates and distributes new certificates and keys to all the nodes,
+* restarts the ``flocker-control`` on the first ``managed`` node, and
+* restarts the ``flocker-*-agent`` processes on all the nodes.
+
+This means that the ``managed`` provider can be used to quickly test different package versions and packages built from different branches.
+
+To use the ``managed`` provider, the configuration file should include:
+
+- **addresses**: A ``list`` of IP addresses of the nodes or a ``dict`` of ``{"<private_address>": "<public_address>"}`` if the public addresses are not configured on the node (see below).
```

Comment:
Thinking forward, I wonder how much more information we'll want to have about nodes. Eventually, perhaps we want 
```yaml
managed:
    nodes:
    - private: 10.x.x.x
      public: 8.8.x.x
      ...
    - ...
```

When I first started thinking about this comment, I was slightly concerned about overriding the meaning of `addresses` and taking over the meaning of a `dict` (and so the new configuration should be `private_address_mapping`. But on further reflection, if we are extending this much further, `nodes` seems like a better key that `addresses` anyway.